src/main/java/storage/FileReaderService.java

```diff
@@ -0,0 +1,8 @@
+package storage;
```

Comment:
```suggestion
package core.basesyntax.service;
```
why are services in storage package?